app/models/jobsModel.js

```diff
@@ -5,18 +5,51 @@ const db = require('../database/dbConfig.js')
  * scheduled_time is an optional field.
  * 
  * @param {"class_id" : integer, 
- *         "scheduled_time" : Date, 
+ *         "scheduled_time" : Date,
  *         "job_hash" : string} job 
  */
-async function insertJob(job){
+async function insert(job){
     try {
         await db('jobs')
-        .insert(job)
-    }catch(e){
-        console.error("Error: failed to insert job", e)
+              .insert(job);
+    } catch(e){
+        console.error("Error: failed to insert job", e);
+    }
+}
+
+/**
+ * Updates scheduled_time for jobs in the database with the same class_id.
+ * 
+ * @param {"class_id" : integer,
+ *         "scheduled_time" : Date} updatedJob 
+ */
+async function update(updatedJob){
+    try {
+        await db('jobs')
+              .where({"class_id" : updatedJob.class_id})
+              .update({"scheduled_time" : updatedJob.scheduled_time});
```

Comment:
since this is an update function, you should just put the entire json into the field so it checks all fields for if anything needs to be updated

`.update(updatedJob)`